samples/Grover/Reflections.qs

```diff
@@ -0,0 +1,67 @@
+namespace GroverSample {
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Convert;
+    open Microsoft.Quantum.Math;
+    open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Arrays;
+    open Microsoft.Quantum.Measurement;
+
+    /// # Summary
+    /// Reflects about the basis state marked by alternating zeros and ones.
+    /// This operation defines what input we are trying to find in the main
+    /// search.
+    operation ReflectAboutMarked(inputQubits : Qubit[], idxMarkedItem : Int) : Unit {
```

Comment:
If we make the change about and remove the call to `ReflectAboutMarked`, we can also remove this operation.